<launch>
    
    <!--- Sim Time -->
    <param name="/use_sim_time" value="false" />

    <!--- Run Rviz-->
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find lili_om)/config/test.rviz" />

    <!-- Parameters -->
    <rosparam file = "$(find lili_om)/config/config_ka_urban_schloss_2.yaml"/>

    <node pkg="tf" type="static_transform_publisher" name="lili_om" args="0 0 0 0 0 0 base_link lili_om 50"/>

    <!--- lili_om -->
    <node pkg="lili_om" type="FormatConvert" name="FormatConvert" output="screen"/>
    <node pkg="lili_om" type="Preprocessing" name="Preprocessing" output="screen"/>
    <node pkg="lili_om" type="LidarOdometry" name="LidarOdometry" output="screen"/>
    <node pkg="lili_om" type="BackendFusion" name="BackendFusion" output="screen"/>

</launch>
